scripts/install-chef-suse.sh

```diff
@@ -911,7 +911,7 @@ fi
 
 # Use current time zone
 (
-    . /etc/sysconfig/clock || : Ignoring missing /etc/sysconfig/clock
+    TIMEZONE=$(LC_ALL=C timedatectl status | grep "zone:" | cut -d: -f2 | cut -d' ' -f2)
```

Comment:
@aspiers sorry for being the target of my frustration here, I didn't mean it that way. I agree the awk suggestion is visually most appealing, but there are a million ways of achieving the same thing here (including writing a erlang program for extracting the time zone), its just not relevant for me in the level of problems that I currently have to deal with.